ktor-features/ktor-auth/jvm/src/io/ktor/auth/OAuth.kt

```diff
@@ -79,7 +79,9 @@ public sealed class OAuthServerSettings(public val name: String, public val vers
         public val accessTokenUrl: String,
 
         public val consumerKey: String,
-        public val consumerSecret: String
+        public val consumerSecret: String,
+
+        public val accessTokenInterceptor: HttpRequestBuilder.() -> Unit = {}
```

Comment:
The purpose of this callback is not clear to the end-users: when it will be called and why I need to pass it.

Reply:
btw It changes public API and should be for `1.5.0`

Reply:
Yes, this is for tuning request and people usually shouldn't use it unless something unusual happens. I would introduce a switch instead, but how many switches do we need? And also disabling status parameter is potentially dangerous in terms of security so not sure if it should be cheap in spite of that it is Dropbox that violates policy.